src/qt/intro.cpp

```diff
@@ -139,17 +140,26 @@ Intro::Intro(QWidget *parent, int64_t blockchain_size_gb, int64_t chain_state_si
     );
     ui->lblExplanation2->setText(ui->lblExplanation2->text().arg(PACKAGE_NAME));
 
+    const int min_prune_target_GB = std::ceil(MIN_DISK_SPACE_FOR_BLOCK_FILES / 1e9);
```

Comment:
nit: why here and after not use https://github.com/bitcoin-core/gui/blob/fa2a5b8f3aeb978b64189203cbbe95a00ac92a6a/src/qt/guiconstants.h#L52-L53
?